ClangSharp.PInvokeGenerator/PInvokeGenerator.cs

```diff
@@ -786,6 +792,30 @@ private string GetParmModifierForPointeeType(Decl decl, Type pointeeType)
             }
         }
 
+        private string GetRemappedCursorName(NamedDecl namedDecl)
```

Comment:
There are both upsides and downsides to the super trivial approach here.

The upside is you can remap literally any name. The downside is that it impacts anything with that name and you may want a more precise filter (such as based on whether it is a "struct" or a "field").